src/commonMain/kotlin/kotools/types/text/NotBlankString.kt

```diff
@@ -64,6 +66,7 @@ public value class NotBlankString private constructor(
 /** Concatenates this string with the [other] one. */
 @ExperimentalSinceKotoolsTypes("4.2")
 @ExperimentalTextApi
+@JvmName("plusOnNotBlankString")
 public operator fun NotBlankString.plus(other: String): NotBlankString = "$this"
     .plus(other)
     .toNotBlankString()
```

Comment:
This function could include an example in its documentation for calling it from Java code.